ietf/templates/submit/tool_instructions.html

```diff
@@ -60,39 +60,39 @@ <h3 class="mt-4">Upload screen</h3>
             </tr>
         </tbody>
         <tbody>
-            <tr>
-                <th scope="row">.txt format</th>
-                <td>Button to select a plain-text file of an I-D from a user's local file system.</td>
-            </tr>
             <tr>
                 <th scope="row">.xml format</th>
                 <td>Button to select an XML file of an I-D from a user's local file system.</td>
             </tr>
+            <tr>
+                <th scope="row">.txt format</th>
+                <td>Button to select a plain text file of an I-D from a user's local file system.</td>
+            </tr>
             <tr>
                 <th scope="row">.pdf format</th>
                 <td>Button to select a PDF file of an I-D from a user's local file system.</td>
             </tr>
             <tr>
                 <th scope="row">Upload</th>
                 <td>
-                    Button to upload the document(s). The tool will begin parsing the plain-text document (or creating it from the xml if only xml is provided) and validate the document. The parsed meta-data will be displayed for user confirmation along with the validation results.
+                    Button to upload the document(s). The tool parses the .xml file or the .txt file, and the metadata is displayed for your confirmation along with the validation results.
                 </td>
             </tr>
         </tbody>
     </table>
     <h3 class="mt-4">Validation screen</h3>
     <p>
-        After a user uploads the document(s),  the tool will parse the plain-text version, validate the I-D, and display the validation results with option(s) for next steps. The validation includes: checking for all IPR-related notices and I-D boilerplate described in
-        <a href="https://www.ietf.org/ietf/1id-guidelines.html">Guidelines to Authors of Internet-Drafts</a>;
+        After you upload the file(s) and the tool parses the the metadata, the validation results are displayed, and you are offered option(s) for next steps. The validation checks include for IPR-related notices and I-D boilerplate described in
+        <a href="https://www.ietf.org/ietf/1id-guidelines.html">Guidelines to Authors of Internet-Drafts</a>,
         the required sections described in
-        <a href="https://www.ietf.org/ID-Checklist.html">the I-D Check List</a>;
-        the version number; and the creation date.
+        <a href="https://www.ietf.org/ID-Checklist.html">the I-D Check List</a>,
+        the version number, and the creation date.
```

Comment:
I wonder if these should be pointing into the authors wiki instead?

Reply:
I guess that is a question for Greg Wood.

Reply:
Those URLs all redirect now to https://authors.ietf.org/en/content-guidelines-overview.